### PR TITLE
bookings controller and root creates

### LINKED ISSUE #65: Booking controller and routing
We need a create action for the booking, that's the only action we need right now.

### PR DESCRIPTION
**Pull request by AntonioDannemann**
Created the root and the controller with only the create method

### CODE REVIEW

File: app/controllers/arts_controller.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class ArtsController < ApplicationController
-  skip_before_action :authenticate_user!, only: %i[index show]
+  before_action :authenticate_user!, only: %i[index show]
 
   def index
     @arts = Art.all
```
Comment (wJoenn) [inline app/controllers/arts_controller.rb]: This should not have been touched.
Changed it back to what it was please

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class BookingsController < ApplicationController
+  def create
+    @booking = Booking.art(bookings_params)
+    @booking.user = current_user
+    if Booking.save
+      redirect_to art_path(@booking.art)
+    else
+      render 'new', status: :unprocessable_entity
+    end
+  end
```
Comment (wJoenn) [inline app/controllers/bookings_controller.rb]: That's not how you create a booking.
booking = Booking.new(params)
booking.user = current_user
booking.art = Art.find(params[:art_id])

Also there are no new so you can't render it. If it's not saved you just redirect to art path too

**Review by wJoenn — CHANGES_REQUESTED** [review]

File: app/controllers/bookings_controller.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+class BookingsController < ApplicationController
+  def create
+    @booking = Booking.new(bookings_params)
+    @booking.user = current_user
+    @booking.art = Art.find(params[:art_id])
+    if Booking.save
+      redirect_to art_path(@booking.art)
+    end
```
Comment (wJoenn) [inline app/controllers/bookings_controller.rb]: Remove the if too

Booking.save
redirect_to

File: app/controllers/bookings_controller.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  end
+
+  private
+
+  def bookings_params
+    params.require(:booking).permit(:start_date, :end_date, :address)
+  end
```
Comment (wJoenn) [inline app/controllers/bookings_controller.rb]: By convention thgis should be booking_params, not bookingS_params

**Review by wJoenn — CHANGES_REQUESTED** [review]

**Review by wJoenn — APPROVED** [review]

**Review by Glod33 — APPROVED** [review]

**Pull request merged**